tests/models/graphormer/test_modeling_graphormer.py

```diff
@@ -460,15 +460,15 @@ def test_for_graph_classification(self):
     @slow
     def test_model_from_pretrained(self):
         for model_name in GRAPHORMER_PRETRAINED_MODEL_ARCHIVE_LIST[:1]:
-            model = GraphormerForGraphClassification.from_pretrained(model_name)
+            model = GraphormerForGraphClassification.from_pretrained(model_name, num_classes=1)
```

Comment:
The default value for `num_classes` is `2` in `GraphormerConfig`. The current fix works, but it means anyone want to load the checkpoint for `GraphormerForGraphClassification` on `clefourrier/graphormer-base-pcqm4mv1/v2` will need to specify `num_classes=1` - otherwise the loading will fail.

This is not good, and I believe the right fix should be updating the config files on the checkpoints repo to have `num_classes=1`.

Let me know if I miss anything :-)

Reply:
After thinking about it a bit, it makes sense, I'll edit it!

Reply:
Hmm, what I mean is to update the files 

- https://huggingface.co/clefourrier/graphormer-base-pcqm4mv1/blob/main/config.json
- https://huggingface.co/clefourrier/graphormer-base-pcqm4mv2/blob/main/config.json

and keep the code not touched.

Let's hear what @sgugger  says though.

Reply:
Sorry, read your comment too fast

Reply:
Let's merge it as such!